tests/sample/unsafe/invalid_maps2.c

```diff
@@ -54,7 +54,7 @@ struct bpf_map_def dummy_map = {
     .key_size = sizeof(uint32_t),
     .value_size = sizeof(uint32_t),
     .max_entries = 1000,
-    .pinning = PIN_GLOBAL_NS};
+    .pinning = PIN_BY_NAME};
```

Comment:
Why can't these use "LIBBPF_PIN_BY_NAME" instead of "PIN_BY_NAME"?

Reply:
Changed it to now use LIBBPF_PIN_BY_NAME